docs/REST API/rest-api.md

```diff
@@ -5494,6 +5495,11 @@ Returns a list of all the labels in Fleet.
 }
 ```
 
+###### Additional notes
+
+When `include_host_counts` is `true` (or omitted), `host_count` will only be included for `labels` that are in use by one or more hosts, but `count` will always be included, even if it is `0`. When `include_host_counts` is `false`, `host_count` will always be omitted, and `count` will be returned as `0` for each label. Setting `include_host_counts=false` will improve API performance, especially on deployments with large numbers of hosts and labels.
+
```

Comment:
Let's cut the heading and move these notes right under the parameters table. (That's the convention elsewhere in the API docs)